### ISSUE TITLE
Progressive download on byte-range segmented webm file

### ISSUE THREAD

**Issue by zerefel** [flowplayer/flowplayer-mpegdash#19]
Hey, it's me again.

I ran into a problem where the entire webm file is downloaded before playback can begin. I am using this DASH playlist for testing purposes: https://s3.eu-central-1.amazonaws.com/overpro.zencoder.eu/B1615JjGZ/dash/B1615JjGZ.mpd

Flowplayer version: **Commercial 7.0.2**
Flowplayer DASH.js version: **latest commit in GitHub**
Relevant part of Flowplayer config: 
```
            dash: {
                dashQualities: true,
                qualitiesForSafari: true,
                dashType: 'video/webm',
                initialVideoQuality: 3
            },
            clip: {
                sources: [
                    {type: 'application/dash+xml', src: 'https://s3.eu-central-1.amazonaws.com/overpro.zencoder.eu/B1615JjGZ/dash/B1615JjGZ.mpd'}
                ],
                title: this.videoTitle
            }
```

I also tried to remove the `initialVideoQuality: 3` setting but that did not help, it just made the player download (again - progressively) another video quality setting, for instance 480p.

I expect the stream to work similarly as in this demo (same manifest used):

http://dash-mse-test.appspot.com/dash-player.html?url=https%3A%2F%2Fs3.eu-central-1.amazonaws.com%2Foverpro.zencoder.eu%2FB1615JjGZ%2Fdash%2FB1615JjGZ.mpd&autoplay=on&adapt=auto&flavor=

**Comment by blacktrash** [comment]
@zerefel - as mentioned previously:
https://github.com/flowplayer/flowplayer-mpegdash/issues/18#issuecomment-304413046
https://github.com/flowplayer/flowplayer-mpegdash#stream-compatibility

See also: https://github.com/Dash-Industry-Forum/dash.js/issues/2044

dash-mse-test.appspot.com uses a different library (and has different issues).

As far as I can see the plugin works if you don't enforce WEBM.

**Comment by zerefel** [comment]
@blacktrash 
True. It seems like Dash.js does not like WebM very much. I did some research on my end and it seems like there are very few players that support WebM-DASH. Not only that but they're also very barebone. Bummer. 

JWPlayer's implementation seems to be able to switch the underlying provider to Shaka, which enables WebM-DASH to be played back properly. 

Given that this is a problem in the Dash.js plugin, you can close this ticket if you feel like nothing could be done. I will wait for the plugin to mature before switching my platform to WebM-DASH as the main video format. 
Thanks for the snappy support once again.

**Comment by arpu** [comment]
no problem with webm-dash  vp9 video codec and opus audio with dash.js 2.5.1 (using videojs)

**Comment by zerefel** [comment]
@arpu How have you segmented your files? Can you link a sample mpd that you use?

**Comment by blacktrash** [comment]
@arpu - will try that; however for mpegdash and VOD there's currently a showstopper with dash.js 2.5.x: https://github.com/Dash-Industry-Forum/dash.js/issues/1901
